<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasterEditWidget</class>
 <widget class="QWidget" name="MasterEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="loMaster">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="lblThisTestDependsOn">
       <property name="text">
        <string>This test depends on </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbDependencyMode">
       <item>
        <property name="text">
         <string>another test(s)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Expression</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDependenciesHideLeft">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>13</width>
         <height>13</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../../qHostMonitor.qrc">
         <normaloff>:/img/minus.gif</normaloff>
         <normalon>:/img/plus.gif</normalon>:/img/minus.gif</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>13</width>
         <height>13</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frmMasterTests">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stwMasterTest">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="SDPO::MasterTestsEditWidget" name="wMasterTests"/>
        <widget class="SDPO::ExpressionTestsEditWidget" name="wExpressionTests"/>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkSynchronizeCounters">
          <property name="text">
           <string>Synchronize counters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkSynchronizeStatusAlerts">
          <property name="text">
           <string>Synchronize status &amp;&amp; alerts</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SDPO::MasterTestsEditWidget</class>
   <extends>QWidget</extends>
   <header>testedit/qMasterTestsEditWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SDPO::ExpressionTestsEditWidget</class>
   <extends>QWidget</extends>
   <header>testedit/qExpressionTestsEditWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
